bedrock/firefox/templates/firefox/firstrun/firstrun-horizon.html

```diff
@@ -55,8 +55,10 @@ <h2 class="larger">{{ _('Sign in to Firefox and you’re good to go.') }}</h2>
           {% endif %}
         </header>
         <div class="fxaccounts-container">
+          {# Bug 1354710: firstrun pages need to pass funnelcake parameters to FxA #}
+          {% set utm_source = 'firstrun' if not funnelcake_id else 'firstrun_f' + funnelcake_id %}
```

Comment:
I wonder if it's worth asking in the bug if this change should remain indefinitely, or should be removed after this round of funnelcakes. I'm guessing the former, but 🤷‍♂️.